features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -772,6 +772,43 @@ URL - [https://ik.imagekit.io/demo/tr:h-700,w-700,cm-pad\_extract,bg-A94D34/medi
 {% endtab %}
 {% endtabs %}
 
+**Blurred background**
+
+Syntax: `blurred[_<BlurIntensity>][_<Brightness>]`
+
+| Optional parameter |                    Possible value |                                                                         Description |   Default |
```

Comment:
We shouldn't write optional parameter here, it should be explained in syntax description. Just write `Parameter` and `Values`.